src/styles/styles.scss

```diff
@@ -0,0 +1,14 @@
+.video-responsive {

```

Comment:
try for a better name than `styles.scss`. How does this differ from youtubevideo.scss